packages/fuel-indexer-tests/src/fixtures.rs

```diff
@@ -160,16 +270,15 @@ pub fn get_test_contract_id() -> Bech32ContractId {
     Bech32ContractId::from(id)
 }
 
-pub async fn api_server_app_postgres() -> Router {
+pub async fn api_server_app_postgres(database_url: Option<&str>) -> Router {
+    let database: DatabaseConfig = if database_url.is_some() {
+        DatabaseConfig::from_str(database_url.unwrap()).unwrap()
+    } else {
+        DatabaseConfig::default()
+    };
```

Comment:
if this is not set `export DATABASE_URL="postgres://postgres:my-secret@localhost:5432"` the password will be set from the default const value `"postgres"` and I think an error will be thrown.  Maybe we should just hardcode a pw for the ephemeral testdb?

Reply:
This makes sense.